### ISSUE TITLE
we need to manufacture a major privacy breach and/or user data scandal in a major social network to get users on sweet

### ISSUE THREAD

**Issue by lowercasename** [lowercasename/sweet#103]
i've decided partisan tactics are the only way

**Comment by toBeOfUse** [comment]
Yeah, I like this idea. I'm also thinking of turning our unfathomable bugs into guerrilla marketing by posting questions on stackoverflow about them and including links at which one can see this behavior on the actual site
